packages/code-studio/src/log/LogExport.ts

```diff
@@ -6,6 +6,48 @@ import { logHistory } from './LogInit';
 
 const FILENAME_DATE_FORMAT = 'yyyy-MM-dd-HHmmss';
 
+// Blacklist applies to all keys (e.g. blacklist foo will blacklist foo, a.foo, and b.foo)
+const KEY_BLACKLIST: string[] = ['client'];
```

Comment:
I don't think we want a `KEY_BLACKLIST` at all, certainly not with something as general as `client`. If it was something like `__keyValue` or a key that is clearly intended to be private, perhaps, but this blacklisting is a bit broad I think. Was there a reason to do a KEY blacklist instead of just by path?